packages/edit-site/src/components/style-book/color-examples.tsx

```diff
@@ -9,16 +9,22 @@ import clsx from 'clsx';
 import { __experimentalGrid as Grid } from '@wordpress/components';
 import { View } from '@wordpress/primitives';
 import {
+	// @ts-ignore
 	getColorClassName,
+	// @ts-ignore
 	__experimentalGetGradientClass,
+	// @ts-ignore
```

Comment:
This is pretty dodgy. Is there a better way @sirreal?

Reply:
Which files do you mean? `color-examples.tsx`?

I'm just trying to silence the following TS warning, and prevent the build error:

`TS2307: Cannot find module @wordpress/ block-editor or its corresponding type declarations.`

Similar to the fields package [example](https://github.com/WordPress/gutenberg/blob/ada9c414381672495cdef851869529ccdab3de72/packages/fields/src/fields/template/template-edit.tsx#L15-L15).

Reply:
Yes, I meant files like this one where `@ts-ignore` or `@ts-expect-error` are used to ignore untyped modules. It's a different tradeoff and the approach taken here is fine.